src/proxy/http2/HTTP2.cc

```diff
@@ -600,6 +602,29 @@ Http2::init()
     Metrics::Counter::createPtr("proxy.process.http2.max_concurrent_streams_exceeded_in");
   http2_rsb.max_concurrent_streams_exceeded_out =
     Metrics::Counter::createPtr("proxy.process.http2.max_concurrent_streams_exceeded_out");
+  http2_rsb.data_frames_in          = Metrics::Counter::createPtr("proxy.process.http2.data_frames_in"),
```

Comment:
Why use these intermediares? Could this be done using a `Metric` span and indexing?

Reply:
Don't we need to have them in http2_rsb?